src/ansys/tools/installer/uninstall.py

```diff
@@ -208,6 +208,13 @@ def _remove_all_installed_python(self):
         for path in find_miniforge_linux(ansys_manager_installed_only=True):
             shutil.rmtree(path, ignore_errors=True)
 
+    def _remove_all_installed_python(self):
+        for path in find_ansys_installed_python_linux():
+            path = path.split("bin")[0]
+            shutil.rmtree(path, ignore_errors=True)
+        for path in find_miniforge_linux(ansys_manager_installed_only=True):
+            shutil.rmtree(path, ignore_errors=True)
+
```

Comment:
Why has this been added and where is it called?

Reply:
My bad, the function has been declared already.